internal/process/process.go

```diff
@@ -27,16 +27,23 @@ func ActivationPID(cfg Configurable) int32 {
 
 	procInfoErrMsgFmt := "Could not detect process information: %v"
 
+	seen := map[int32]bool{}
```

Comment:
These integration tests triggered an issue on Windows with PID recycling. The loop below would continue forever until the tests timed out. I can remove these fixes and file a blocking story if we want to discuss the solution.

Reply:
Nice catch! But yeah let's track this via its own story/PR. If for no other reason than that it will make it easier to keep track of the "why" on these code changes.